### PR TITLE
Email only on prod for staffing

### PR DESCRIPTION
**Pull request by kkamalov**

### CODE REVIEW

**Comment by joshblum** [comment]
why don't we just use this backend if it's not production? https://docs.djangoproject.com/en/1.9/topics/email/#console-backend

File: orchestra/bots/staffbot.py [hunk 1]
```diff
@@ -194,11 +194,12 @@ def _get_staffing_request_message(self, staffing_request, template):
     def _send_staffing_request_by_mail(self, staffing_request, message):
         email = (
             staffing_request.communication_preference.worker.user.email)
-
+        logger_only = not settings.PRODUCTION
```
Comment (marcua) [inline orchestra/bots/staffbot.py]: pause :)

Comment (marcua) [inline orchestra/bots/staffbot.py]: `settings.ORCHESTRA_SEND_STAFFING_MESSAGES`

**Comment by joshblum** [comment]
lgtm

**Pull request merged**